From fb1763465e8720cd090a9787ef52376611064337 Mon Sep 17 00:00:00 2001
From: Hou Qi <qi.hou@nxp.com>
Date: Fri, 13 Sep 2024 22:18:54 +0900
Subject: [PATCH 01/19] V4L2Device: Correct v4l2 codec device path

Change codec device pattern to /dev/video, and select one
correct device path /dev/videox where x is an integer.

Upstream-Status: Inappropriate [NXP specific]
---
 media/gpu/v4l2/v4l2_device.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/media/gpu/v4l2/v4l2_device.cc b/media/gpu/v4l2/v4l2_device.cc
index 39c16611c76f3..ebbf240a3c5e7 100644
--- a/media/gpu/v4l2/v4l2_device.cc
+++ b/media/gpu/v4l2/v4l2_device.cc
@@ -852,8 +852,8 @@ void V4L2Device::CloseDevice() {
 }
 
 void V4L2Device::EnumerateDevicesForType(Type type) {
-  static const std::string kDecoderDevicePattern = "/dev/video-dec";
-  static const std::string kEncoderDevicePattern = "/dev/video-enc";
+  static const std::string kDecoderDevicePattern = "/dev/video";
+  static const std::string kEncoderDevicePattern = "/dev/video";
   static const std::string kImageProcessorDevicePattern = "/dev/image-proc";
   static const std::string kJpegDecoderDevicePattern = "/dev/jpeg-dec";
   static const std::string kJpegEncoderDevicePattern = "/dev/jpeg-enc";
-- 
2.34.1

